commands/verify/verify.go

```diff
@@ -228,20 +274,33 @@ func DoVerifyAccordingConfig() error {
 
 			for current := 1; current <= retryCount; current++ {
 				if err := verifySingleCase(v.GetExpected(), v.GetActual(), v.Query); err == nil {
+					summary.successNum++
+					_ = spinnerLiveText.Stop()
 					break
 				} else if current != retryCount {
-					logger.Log.Warnf("verify case failure, will continue retry, %v", err)
+					if current == 1 {
+						logger.Log.Warnf("verify case[%d] failure, will continue retry", idx+1)
+					}
+					Msg := fmt.Sprintf("Retrying to verify case[%d]  [%d/%d]", idx+1, current, retryCount)
+					spinnerLiveText.UpdateText(Msg)
 					time.Sleep(interval)
 				} else {
+					summary.errNum++
+					Msg := fmt.Sprintf("Retrying to verify case[%d]  [%d/%d]", idx+1, current, retryCount)
```

Comment:
can you lowercase the local variable names? `Msg` --> `msg`, also check the codes in your previous PR and fix them.

Reply:
OK!